docs/index.md

```diff
@@ -107,14 +118,6 @@ You can also easily perform the installation or upgrade manually if preferred. D
 * Run `ddev` to test your installation. You should see DDEV's command usage output.
 * As a one-time initialization, run `mkcert -install`, which may require your sudo password. Linux users may have to take additional actions as discussed below in [Linux `mkcert -install` additional instructions](#Linux-mkcert--install-additional-instructions). If you don't have mkcert installed, you can install it from <https://github.com/FiloSottile/mkcert/releases>. Download the version for the correct architecture and `sudo mv <downloaded_file> /usr/local/bin/mkcert && sudo chmod +x /usr/local/bin/mkcert`.
 
-### Installation via package managers - Linux
-
-The preferred Linux package manager is [Homebrew](http://brew.sh/) : `brew install drud/ddev/ddev`
-
-We also currently maintain a package on [Arch Linux (AUR)](https://aur.archlinux.org/packages/ddev-bin/)
-
-As a one-time initialization, run `mkcert -install`, which may require your sudo password. See below for additional information.
-
```

Comment:
It would be a huge win to include a ddev package for Debian-based and RedHat-based systems too.

Reply:
I would dearly love to see that, see https://github.com/drud/ddev/issues/1678 - contribution would be welcome. There are some not-too-hard ways to do it.